src/Check/UsersBlockedNumberOne.php

```diff
@@ -58,6 +58,9 @@ public function getResultInfo() {}
    * {@inheritdoc}.
    */
   public function getResultPass() {
+    if ($this->registry->vendor == "pantheon") {
+      return $this->t('UID #1 is not blocked, but --vendor=pantheon exempts this test.');
```

Comment:
We shouldn't reference `--vendor` or `pantheon` here, as these messages are user-visible in the status tab. We should consider the test a pass regardless of the status of UID #1 on Pantheon, but we should also report the correct blocked / unblocked state. This code looks like it reports that UID #1 is not blocked, even if it is.